packages/runtime/container-runtime/src/test/opLifecycle/OpGroupingManager.spec.ts

```diff
@@ -24,7 +24,9 @@ describe("OpGroupingManager", () => {
 		opHasMetadata: boolean = false,
 		batchId?: string,
 	): IBatch => ({
-		...messagesToBatch(new Array(length).fill(createMessage(opHasMetadata, batchId))),
+		...messagesToBatch(
+			Array.from({ length }).map(() => createMessage(opHasMetadata, batchId)),
```

Comment:
Interesting suggestion from the rule... doesn't this end up creating two arrays (and discarding the first one, the result of `Array.from({ length })` instead of one and then acting on it? So causing a bit more memory churn?

Reply:
Yeah, seems like this would be a bit better?
```typescript
Array.from({ length }, () => createMessage(opHasMetadata, batchId));
```

There's another instance of this a couple of files down in the Github files view.

Reply:
Ah, I actually didn't realize `from` had a transformation option. I'll give that a test. `fill` isn't strongly typed, so I couldn't use it without silly casts, but the `from` overload is probably nicer.